.travis.yml

```diff
@@ -1,38 +1,115 @@
-language: c
-sudo: false
+# Automatically generated by nengo-bones, do not edit this file directly
+# Version: 0.1.0.dev0
+
+language: python
+python:
+  - "3.6"
 notifications:
   email:
-    - tbekolay@gmail.com
+    on_success: change
+    on_failure: change
+# cache:
+#   directories:
+#   - $HOME/miniconda
+
+dist: trusty
 
 env:
   global:
-    - PYTHON="3.5.2"
+    - SCRIPT="test"
+    - PYTHON_VERSION="3.6"
+    - PIP_UPGRADE="true"  # always upgrade to latest version
+    - PIP_UPGRADE_STRATEGY="eager"  # upgrade all dependencies
+    - TEST_ARGS=""
+    - COV_CORE_SOURCE=  # early start pytest-cov engine
+    - COV_CORE_CONFIG=.coveragerc
+    - COV_CORE_DATAFILE=.coverage.eager
+    - PYTHON_VERSION="3.5.2"
     - NENGO_VERSION="nengo"
 
-matrix:
+jobs:
   include:
-    - stage: basic
-      env: MODE="emulator"
-    - env: MODE="static"
-    - stage: advanced
-      env: MODE="emulator"
-           NENGO_VERSION="git+https://github.com/nengo/nengo.git"
-    - env: MODE="hardware"
-           NENGO_VERSION="git+https://github.com/nengo/nengo.git"
-    - env: MODE="docs"
-           NENGO_VERSION="git+https://github.com/nengo/nengo.git"
+  -
+    env:
+      SCRIPT="emulator"
+    stage: basic
+  -
+    env:
+      SCRIPT="static"
+  -
+    env:
+      SCRIPT="emulator"
+      NENGO_VERSION="git+https://github.com/nengo/nengo.git#egg=nengo"
+    stage: advanced
+  -
+    env:
+      SCRIPT="hardware"
+      NENGO_VERSION="git+https://github.com/nengo/nengo.git#egg=nengo"
+  -
+    env:
+      SCRIPT="docs"
+      NENGO_VERSION="git+https://github.com/nengo/nengo.git#egg=nengo"
 
 before_install:
-  - source .ci/conda.sh install
+  # export travis_terminate for use in scripts
+  - export -f travis_terminate
+  - export -f _travis_terminate_linux
+  - export -f _travis_terminate_osx
+  - export -f _travis_terminate_unix
+  - export -f _travis_terminate_windows
+  # install/run nengo-bones
+  - pip install git+https://github.com/nengo/nengo-bones.git@docs
```

Comment:
NTS: This is what's currently causing the build to fail, because `docs` is not rebased onto `template-ci` (but `docs-cleanup` is).

Reply:
`docs` is now rebased onto `template-ci` and `docs-cleanup` deleted